### ISSUE TITLE
Can't Uninstall Mac OS Theme

### ISSUE THREAD

**Issue by slashtechno** [Botspot/pi-apps#419]
I was tried to uninstall Mac OS Theme because the installed failed. I need to uninstall it but I get this error

```
Uninstalling Mac OS Theme
/home/pi/pi-apps/apps/Mac OS Theme/uninstall: line 11: /home/pi/.local/share/MacOSBigSurThemeConverter/uninstall.sh: Permission denied

Failed to uninstall Mac OS Theme!

Close this window to exit.



```

**Comment by Itai-Nelken** [comment]
@techcoder20

**Comment by Itai-Nelken** [comment]
I made a pr that should fix it, #420

**Comment by techcoder20** [comment]
That is strange. The install script makes the uninstall script  executable so I didn't see the need to add it in the uninstall script.
